setup.py

```diff
@@ -65,8 +65,8 @@ class Extension(extension.Extension, object):
 ext = '.pyx' if USE_CYTHON else '.cpp'
 
 extensions = [Extension(
-    name="minorminer_c",
-    sources=["./python/minorminer_c" + ext],
+    name="_minorminer",
```

Comment:
I think you just need to change this to `minorminer/_minorminer` to fix the tests